source/set_up_3dsecure/index.html.md.erb

```diff
@@ -9,12 +9,14 @@ weight: 170
 
 Strong Customer Authentication (SCA) makes online payments more secure for users and reduces fraud. SCA is part of the Payment Service Directive 2 (PSD2).
 
-If you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein, you must implement SCA by 31 December 2020.
+If you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein, you must implement SCA when you [go live](/switching_to_live/#go-live).
 
 If you only take payments within the UK, you must implement SCA by 14 September 2021.
 
 To implement SCA, you must set up 3D Secure 2 (3DS2) or 3D Secure 1 (3DS1) payment authentication.
 
+Payment service providers (PSPs) recommend setting up 3DS2, because they'll start retiring 3DS1 in 2022.
```

Comment:
I think the second clause should be more along the lines of “because 3DS1 will start to be retired in 2022” to reflect it’s the card schemes driving this.

Reply:
Unfortunately "3DS will start to be retired in 2022" is passive voice - can we find an active version by adding the responsible party? From the conversation above - would "because card companies will start to retire..." be ok?

Reply:
Microsoft Word always used to tell me I use the passive voice too much. That and my sentences are too long.

I think the term to use is probably “card schemes” and adding in the examples of Visa and Mastercard is probably necessary because a lot of people won’t know what card schemes are.

Reply:
Thanks - updated in https://github.com/alphagov/pay-tech-docs/pull/541/commits/0f54015cde7a8a077f06846065a3cb8624cc69e2.

Reply:
p.s. passive vs active is hard! It's so easy to use it accidentally. 2iers often spot it in my own content where I've missed it.